### DISCUSSION TITLE
MONAI Invitation: Modeling and Simulation in Medicine Workshop at ANNSIM 2025

### DISCUSSION THREAD

**Discussion by michelodu** [Project-MONAI/MONAI#8262] [General]
Dear colleagues, this message serves to invite you to submit a paper to the upcoming Modeling and Simulation in Medicine Workshop, within the 2025 ANNSIM Modeling & Simulation conference.

2025 ANNUAL MODELING AND SIMULATION CONFERENCE (ANNSIM) - annsim.org
May 26-29, 2025 Complutense University of Madrid, Spain
MODELING AND SIMULATION IN MEDICINE WORKSHOP
Chairs: Prof. Michel Audette, Old Dominion University
Prof. Jerzy Rozenblit, University of Arizona

Dates
Paper submission Jan 19, 2025
Acceptance notification (papers) March 7, 2025
Camera-ready version March 26, 2025
Conference Program Announced April 9, 2025

Topics
The conference will focus on (but is not limited to) the following topical areas.
Modeling and simulation in medicine: fundamental research
Training and education
Care delivery, outcomes, and patient safety
Robotics and its applications in training and in vivo
Life-critical systems
Systems integration: connected health
Cybersecurity and healthcare
Human performance and ergonomics models
Discrete-event and agent-based approaches to healthcare
Physiological modeling, from single-system to integrative approaches
Multi-scale modeling and simulation in medicine
Descriptive anatomical modeling for medical simulation

**Comment by tushar-c** [comment]
Hi, glad to see the upcoming conference. I work in the Medical Technology Industry (www.medprax.in) and is there an option to attend the conference and see the submissions remotely? Thanks and Regards.
